app/views/likes/shared/_show.html.erb

```diff
@@ -0,0 +1,11 @@
+<div>
+  Likes: <%= object.likes.count %>
```

Comment:
- [ ] **[OPTIONAL]** `object.likes.count` will perform queries to the database as it will count the number of likes columns for every post. As your application grows this will slow down your application so you need another mechanism to count the likes, here comes the `counter_cache` which basically cache the `count` and save it into separate column in `posts`, or `comments` table.
[1](https://blog.appsignal.com/2018/06/19/activerecords-counter-cache.html) [2](https://stackoverflow.com/questions/56913688/how-to-get-the-size-of-belong-to-record/56916353#56916353)

Reply:
Thank you very much for this great advice. We will implement this in the next branch.